src/extensions/default/JavaScriptCodeHints/ParameterHintManager.js

```diff
@@ -108,7 +108,11 @@ define(function (require, exports, module) {
         left = Math.max(left, editorLeft);
         left = Math.min(left, editorLeft + $editorHolder.width() - hintWidth);
 
-        if (top < 0) {
+        // Get the top of the editable document
+        var cm = session.editor._codeMirror;
+        var localTop = cm.charCoords({line: 0, ch: 0}).top;
```

Comment:
This won't be correct if the document has been scrolled.

What you want is to compare `top` to the top bounds of the editor, which is probably something like `$editorHolder.offset().top`.  That ignores split views and looks at the bounds of the entire "main view" content area, but the z-ordering probably works out such that it's ok to overlap a neighboring split-view pane... worth double checking with testing though.